app/assets/stylesheets/application.css.scss

```diff
@@ -16,15 +16,33 @@
 
  */
 
+body {
+  background-color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.